llvm/lib/Target/AArch64/AArch64FastISel.cpp

```diff
@@ -2516,6 +2516,10 @@ bool AArch64FastISel::selectIndirectBr(const Instruction *I) {
   if (AddrReg == 0)
     return false;
 
+  // Authenticated indirectbr is not implemented yet.
```

Comment:
It looks like that a test with FastISel ensuring that we fall back to SelectionDAG ISel is missing. I'm OK with adding missing tests later as a separate patch.